breathecode/feedback/tasks.py

```diff
@@ -110,31 +115,36 @@ def new_answer(answer: Answer):
         answer = Answer(cohort=survey.cohort, academy=survey.cohort.academy, lang=survey.lang)
         _answers.append(new_answer(answer))
 
-        # ask for each teacher, with a max of 2 teachers
-        count = 0
-        for ct in cohort_teacher:
-            if count >= survey.max_teachers_to_ask:
-                break
-            answer = Answer(mentor=ct.user, cohort=survey.cohort, academy=survey.cohort.academy, lang=survey.lang)
-            _answers.append(new_answer(answer))
-            count = count + 1
-
-        # ask for the first TA
-        cohort_assistant = CohortUser.objects.filter(
-            cohort=survey.cohort, role="ASSISTANT", educational_status__in=["ACTIVE", "GRADUATED"]
-        )
-        count = 0
-        for ca in cohort_assistant:
-            if count >= survey.max_assistants_to_ask:
-                break
-            answer = Answer(mentor=ca.user, cohort=survey.cohort, academy=survey.cohort.academy, lang=survey.lang)
-            _answers.append(new_answer(answer))
-            count = count + 1
+        if not survey.cohort.available_as_saas:
```

Comment:
`cohort.available_as_saas` could be None in this case it uses `cohort.academy.available_as_saas`

Reply:
```py
        if survey.cohort.available_as_saas is False or (survey.cohort.available_as_saas is None and survey.cohort.academy.available_as_saas):
```